4/run-docker-silo-cluster.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+docker-compose build &&
+  docker-compose up
```

Comment:
Let's also provide and accurately describe convenience scripts illustrating more exotic (or maybe even all!) types of communication:

* local client1, dockerized client2 - local cluster of silos
* local client1, local client2 - dockerized cluster of silos
* local client1, dockerized client2 - one dockerized silo, one local silo
* etc.

People using this repository might have very specific needs and sometimes those things are very tricky to configure.

Now I think I know why you used `Dns.GetHostAddresses` and not the method I used in the previous solutions - it only works in docker :) Let's make sure that whatever we come up with is ready to work in any type of environment.

Reply:
So made a little discovery around an issue that exists with local clients connecting to a docker cluster. It's not technically possible(outside of docker containers sharing ip of machine it's on). There is one exception, a local silo can join a docker cluster. Then a local client is capable of using that gateway. Here's some background info around it. https://github.com/dotnet/orleans/issues/6420